docs/EN/Configuration/xdrip.md

```diff
@@ -0,0 +1,238 @@
+# xDrip+ settings
+
+If not already set up then download [xdrip](https://github.com/NightscoutFoundation/xDrip)
+
+For G6 transmitters manufactured after fall/end of 2018 please make sure to use one of the [latest xDrip+ versions](https://github.com/NightscoutFoundation/xDrip/releases). Those transmitters have a new firmware and latest stable version of xDrip+ (2019/01/10) cannot deal with it.
+
+## Basic settings for all CGM & FGM systems
+
+* Make sure to set Base URL correctly including <font color="#FF0000"><b>S</font></b> at the end of http<b><font color="#FF0000">s</font></b>:// (not http://)
+
+   i.e. https://API_SECRET@your-app-name.herokuapp.com/api/v1/
+
+   -> Hamburger Menu (top left of homescreen) -> Settings-> Cloud Upload-> Nightscout Sync (REST-API) -> Base URL
+* Deactivate “Automatic Calibration”
+   If the checkbox for "Automatic Calibration" is checked, activate "Download data" once, then remove the checkbox for "automatic calibration" and deactivate "Download data" again, otherwise the treatments (insulin & carbs) will be added twice to Nightscout.
+* Tap "Extra Options"
+* Deactivate "Upload treatments" and "Back-fill data"
+* Option "Alert on failures" should also be deactivated. Otherwise you will get an alarm every 5 minutes in case wifi/mobile network is too bad or the server is not available.
+
+   ![xDrip+ Basic Settings 1](../images/xDrip_Basic1.png)
+
+   ![xDrip+ Basic Settings 2](../images/xDrip_Basic2.png)
+
+* **InterApp-Settings** (Broadcast)
+   If you are going to use a DIY closed loop system and the data should be forwarded to i.e. AndroidAPS you have to activate broadcasting in xDrip+ in Inter-App settings.
```

Comment:
Everybody reading this wants to use AndriodAPS, mostly in CL mode, I guess. So we should be more AAPS specific here

Reply:
Done